style.css

```diff
@@ -0,0 +1,91 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 4910px;
+  width: 320px;
```

Comment:
- [ ] Kindly make sure the width is 375 px as recommended by the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) you must stick to the design :+1: .  In my opinion I would suggest you to use `vw` and percentages to make the website responsive  and cool :smile: